tests/examples/04_assert_fact.yml

```diff
@@ -12,6 +12,8 @@
         assert_fact:
           fact:
             msg: hello world
+            alpha: 1
```

Comment:
Make a new test for this instead of changing one of the examples.  Add the test to test_engine instead of test_examples.